packages/expo-notifications/src/getDevicePushTokenAsync.web.ts

```diff
@@ -31,15 +31,24 @@ function guardPermission() {
 }
 
 async function _subscribeDeviceToPushNotificationsAsync(): Promise<DevicePushToken['data']> {
-  // @ts-ignore: TODO: not on the schema
-  if (!Constants.manifest.notification?.vapidPublicKey) {
+  const vapidPublicKey: string | null =
+    // @ts-ignore: TODO: not on the schema
```

Comment:
```suggestion
    // @ts-expect-error: TODO: not on the schema
```
this way we get alerted once the comment is no longer necessary